lib/internal/structured_clone.js

```diff
@@ -1,12 +1,20 @@
 'use strict';
 
+const {
+  codes: { ERR_MISSING_ARGS },
+} = require('internal/errors');
+
 const {
   MessageChannel,
   receiveMessageOnPort,
 } = require('internal/worker/io');
 
 let channel;
 function structuredClone(value, options = undefined) {
+  if (arguments.length === 0) {
+    throw new ERR_MISSING_ARGS('input');
```

Comment:
```suggestion
    throw new ERR_MISSING_ARGS('value');
```
Better to be consistent with the parameter naming.

Reply:
My bad, copy-pasta is strong with this one 🤦🏻‍♂️ Thanks, good catch